contracts/ManagerProxy.sol

```diff
@@ -0,0 +1,52 @@
+pragma solidity ^0.4.13;
+
+import "./ManagerProxyTarget.sol";
+
+
+contract ManagerProxy is ManagerProxyTarget {
```

Comment:
The proxy contract inherits from the ManagerProxyTarget base contract to ensure that it starts off with the same storage layout as the target contract